tfhe/docs/getting_started/quick_start.md

```diff
@@ -2,11 +2,81 @@
 
 This document explains the basic steps of using the high-level API of **TFHE-rs.**
 
-## Workflow explanation
+## Setting up a Rust project
 
-These are the steps to use the **TFHE-rs** high-level API:
+If you already know how to set-up a Rust project you can go to the next [section](#using-tfhe-rs-and-its-apis).
 
-1. [Import the **TFHE-rs** prelude](quick\_start.md#imports)
+As a first step you need to install the Rust programming language tools. You can do so by visiting https://rustup.rs/ and following instructions there. If you want alternative ways of installing Rust you can check out https://rust-lang.github.io/rustup/installation/other.html for more options.
+
+Once you have Rust installed you should be able to call the build and package manager `Cargo`:
+
+```console
+$ cargo --version
+cargo 1.81.0 (2dbb1af80 2024-08-20)
+```
+
+Your version may vary depending on when you installed Rust. To update your installation you can invoke `rustup update` and it will update your installation of Rust.
+
+You can now invoke `Cargo` to create a new default Rust project:
+
+```console
+$ cargo new tfhe-example
+    Creating binary (application) `tfhe-example` package
+note: see more `Cargo.toml` keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+```
+
+This will create a `tfhe-example` directory and populate it with the following:
+
+```console
+$ tree tfhe-example/
+tfhe-example/
+├── Cargo.toml
+└── src
+    └── main.rs
+
+1 directory, 2 files
+```
+
+You now have a minimal Rust project, in the next section we'll explain how to add **TFHE-rs** as a dependency to the project and start using it to perform FHE computations.
+
+## Using TFHE-rs and its APIs
```

Comment:
In this next section, we'll explain how to add **TFHE-rs** as a dependency to the project and start using it to perform FHE computations.